### PR TITLE
Share modal

### PR DESCRIPTION
**Pull request by matthewthomsonnz**
- This PR replaces the social media icons with `<s72-share-modal ... />
- Makes use of [ this PR/Branch](https://github.com/indiereign/shift72-web/pull/246)
[AB#6477](https://dev.azure.com/S72/SHIFT72/_boards/board/t/Web%20team/Stories/?workitem=6477)

### CODE REVIEW

**Comment by stajs** [comment]
@matthewthomsonnz is this ready for review?

**Comment by matthewthomsonnz** [comment]
> @matthewthomsonnz is this ready for review?

yep! requested now.

**Comment by PeculiarGoat** [comment]
I'm guessing this depends on [this Relish PR](https://github.com/indiereign/shift72-web/pull/246) being merged/released first?

**Review by PeculiarGoat — CHANGES_REQUESTED** [review]
Looks good but the colours should probably match the other modals on the site (light text on dark background).

**Comment by matthewthomsonnz** [comment]
> I'm guessing this depends on [this Relish PR](https://github.com/indiereign/shift72-web/pull/246) being merged/released first?

correct

**Comment by matthewthomsonnz** [comment]
> Looks good but the colours should probably match the other modals on the site (light text on dark background).

Light/dark mode support added

**Comment by stajs** [comment]
@matthewthomsonnz Since we need the Relish PR first, adding a blocked label and converting to draft to prevent accidental early merging.

**Comment by stajs** [comment]
@matthewthomsonnz Okay, turns out you can't review in draft, so will just rely on the label.

File: site/styles/_shopping.scss [hunk 1]
```diff
@@ -13,11 +13,9 @@
 
     .s72-btn-close {
       background: transparent;
-      color: var(--body-color);
       display: block;
       height: 30px;
       line-height: 30px;
-      opacity: 1;
```
Comment (stajs) [inline site/styles/_shopping.scss:20]: Just confirming you've checked the shopping modal is good for light/dark themes?

**Review by stajs — APPROVED** [review]
:shipit:

**Comment by stajs** [comment]
@matthewthomsonnz back to draft now that I've reviewed!

**Comment by stajs** [comment]
@matthewthomsonnz 🤦 actually drafts can be reviewed, but drafts don't have the button on the convo tab.

File: site/styles/_shopping.scss (hunk 1 above)
Comment (matthewthomsonnz) [reply site/styles/_shopping.scss:20]: Sure did and double-checked just then also.

Note the above removed opacity/color now applied to shopping modal close via [this .extend](https://github.com/shift72/core-template/blob/develop/site/styles/_shift72.scss#L26-L28) and [this addition](https://github.com/shift72/core-template/pull/269/files#diff-37c2412e0517de1e975b3f3149c9fec0ee515c03503c64199b5e41ef94ec8103R56-R58)

**Comment by stajs** [comment]
@matthewthomsonnz I've dropped the blocked tag since the Relish PR is merged. I'll let you convert from draft when you are ready.

**Comment by stajs** [comment]
@matthewthomsonnz any chance we can get this complete before your vacay?

**Comment by matthewthomsonnz** [comment]
> @matthewthomsonnz any chance we can get this complete before your vacay?

@stajs Sorry about delay in response. Will push for a relish prod release today, then will merge once deployed. On the odd chance that doesn't happen today I'll get someone in the team to make sure this happens

File: site/styles/_shopping.scss [hunk 2]
```diff
@@ -13,11 +13,9 @@
 
     .s72-btn-close {
       background: transparent;
-      color: var(--body-color);
```
Comment (PeculiarGoat) [inline site/styles/_shopping.scss:16]: Removing this means the other modals have a black X on a black background (just tested it).  I don't think you should remove the colour and opacity lines from this file.

Also, is there a reason why the share modal is light theme when the rest of the modals on the site are dark theme?

Comment (matthewthomsonnz) [reply site/styles/_shopping.scss]: Good find @PeculiarGoat. I've re-added that line and the x has reappeared.

I had pushback from design on this, so had to revert it.

I really preferred dark mode but picking battles.

**Review by PeculiarGoat — APPROVED** [review]
Gonna approve and give a good testing later. :+1:

**Pull request merged**